server/libs/skills/xl/moveCellContent/moveCellContent.js

```diff
@@ -116,44 +116,54 @@ class moveCellContent extends ExcelBaseSkill {
       }
 
     }
-    var resolveParams = {"attrValue" : finalArray};
+    var resolveParams = { "attrValue": finalArray };
     return Promise.resolve(resolveParams);
 
 
   }
 
 
 
-  getSheetNameAndSheetCountFromInitDocJSON(initDocJSON, dependantSheetArrayInModel) {
+  getSheetNameAndSheetCountFromInitDocJSON(initDocJSON, dependantSheetArrayInModel, clonedDependantSheetArrayInModel) {
```

Comment:
fn name should not be that much long!!
suggestion: getSheetDetails() or getSheetNamenCount()